BlueMap/Chat/src/main/java/codes/antti/auth/bluemap/chat/BlueMapChatPlugin.java

```diff
@@ -179,6 +209,24 @@ public void onChatMessage(AsyncPlayerChatEvent event) {
 		forEachSession(sse -> sse.send(message));
 	}
 
+	@EventHandler(priority = EventPriority.MONITOR)
+	public void onServerCommand(ServerCommandEvent event) {
```

Comment:
This only handles console commands but the say command could be invoked by a player too

Reply:
I will do testing but it think `onChatMessage` handles the say command.

Reply:
Nope... it doesn't will fix